Tests/MapboxSearchTests/Common/Models/Metadata/OpenHoursTests.swift

```diff
@@ -3,28 +3,28 @@ import XCTest
 
 class OpenHoursTests: XCTestCase {
     func testAlwaysOpenedCoreConversion() throws {
-        let coreHours = CoreOpenHours(mode: .alwaysOpen, periods: [], weekdayText: nil, note: nil)
+        let coreHours = CoreOpenHours(mode: .alwaysOpen, periods: [], weekdayText: nil, note: nil, exceptionalOpenings: [], exceptionalClosings: [])
         let hours = OpenHours(coreHours)
 
         XCTAssertEqual(hours, .alwaysOpened)
     }
 
     func testTemporarilyClosedCoreConversion() throws {
-        let coreHours = CoreOpenHours(mode: .temporarilyClosed, periods: [], weekdayText: nil, note: nil)
+        let coreHours = CoreOpenHours(mode: .temporarilyClosed, periods: [], weekdayText: nil, note: nil, exceptionalOpenings: [], exceptionalClosings: [])
         let hours = OpenHours(coreHours)
 
         XCTAssertEqual(hours, .temporarilyClosed)
     }
 
     func testPermanentlyClosedCoreConversion() throws {
-        let coreHours = CoreOpenHours(mode: .permanentlyClosed, periods: [], weekdayText: nil, note: nil)
+        let coreHours = CoreOpenHours(mode: .permanentlyClosed, periods: [], weekdayText: nil, note: nil, exceptionalOpenings: [], exceptionalClosings: [])
         let hours = OpenHours(coreHours)
 
         XCTAssertEqual(hours, .permanentlyClosed)
     }
 
     func testScheduledCoreConversion() throws {
-        let coreHours = CoreOpenHours(mode: .scheduled, periods: .coreOpenHourPeriods(), weekdayText: nil, note: nil)
+        let coreHours = CoreOpenHours(mode: .scheduled, periods: .coreOpenHourPeriods(), weekdayText: nil, note: nil, exceptionalOpenings: [], exceptionalClosings: [])
```

Comment:
It looks like we also need to support this feature with exceptionalOpenings/exceptionalClosings in the Search SDK. Could you please check if this feature is blocker for rc release or not?